packages/lodestar/src/api/rest/controllers/debug/beacon/getHeads.ts

```diff
@@ -0,0 +1,26 @@
+import {ListType} from "@chainsafe/ssz";
+import {ApiController} from "../../types";
+
+const MAX_FORKCHOICE_HEADS = 1000;
+
+export const getHeads: ApiController = {
+  url: "/beacon/heads",
+
+  handler: async function (req, resp) {
+    const heads = await this.api.debug.beacon.getHeads();
+    if (!heads) {
+      return resp.status(404).send();
+    }
+    const slotRootsType = new ListType({
+      elementType: this.config.types.SlotRoot,
+      limit: MAX_FORKCHOICE_HEADS,
+    });
```

Comment:
you could just iterate trough heads and serialize each of them without instantiating ListType